force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -29,7 +29,7 @@ public with sharing class FlowControlLoopsCollections {
      * @return "Hello World!" if x greater than y, otherwise return null.
      */
     public static String helloWorld(Integer x, Integer y) {
-        return null; // Replace null with the variable you used to store the result
+        return (x > y) ? 'Hello World!' : (x.equals(y) ? null : null);
```

Comment:
Great job using a ternary expression. You don't actually need the second nested condition though. If x is not greater than y, you can return null.